packages/protocol/contracts/governance/Election.sol

```diff
@@ -475,14 +501,55 @@ contract Election is
    * @return The total number of votes cast by an account.
    */
   function getTotalVotesByAccount(address account) external view returns (uint256) {
-    uint256 total = 0;
+    bool allowedToVoteOverMaxNumberOfGroupsForAccount = allowedToVoteOverMaxNumberOfGroups[account];
     address[] memory groups = votes.groupsVotedFor[account];
+    require(
+      allowedToVoteOverMaxNumberOfGroupsForAccount || groups.length < maxNumGroupsVotedFor,
+      "Too many groups voted for"
+    );
+
+    if (groups.length > maxNumGroupsVotedFor) {
+      uint256 totalVotes = cachedVotesByAccount[account].totalVotes;
+      require(totalVotes > 0, "Total votes are not counted! Use updateTotalVotesByAccount.");
```

Comment:
Is this `require` necessary? 0 votes is a valid value.

Reply:
This whole method is now removed